packages/@aws-cdk/core/test/test.secret-value.ts

```diff
@@ -47,6 +47,11 @@ export = {
     test.done();
   },
 
+  'secretsManager with a non-ARN ID that has colon'(test: Test) {
```

Comment:
I think the message is wrong in the test (it should be `/secret id "not:an:arn" is not an ARN but contains ":"/`).

Reply:
Yup, that was the problem. Fixed.